utilities/crs_dataloader/fetch_localization_ui.ipynb

```
@@ -0,0 +1,1079 @@
+    "                        error_messages.append(f\"      {i}. {msg}\")\n",
+    "                \n",
+    "                if len(sheet_errors) > 3:\n",
+    "                    error_messages.append(f\"      ... and {len(sheet_errors) - 3} more errors\")\n",
+    "            \n",
+    "            return False, error_messages\n",
+    "            \n",
+    "    except ImportError as e:\n",
+    "        # If validator not available, skip validation\n",
+    "        print(f\"   ⚠️  Validator not available: {str(e)}, saved anyway\")\n",
+    "        return True, []\n",
+    "    except Exception as e:\n",
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)